Add 2026-04-20 Unicode string comparison changelog entry (#1947)
inviwo/inviwo: CHANGELOG.md

- **OrthographicCamera**: optional mouse-centered zoom — the point under the cursor stays fixed during scroll zoom

## 2026-04-16 NRRD file format reader
A NRRD (Nearly Raw Raster Data) volume reader has been added to the Base module, enabling loading of `.nrrd` and `.nhdr` files directly into Inviwo.

## 2026-04-16 Portable `util::concat` range adaptor (`std::views::concat` backport)
A new header `include/inviwo/core/util/concat.h` provides a portable implementation of `std::views::concat` (C++26), which is not yet available in Clang/libc++. Only providing forward_iterators.

- **`concat_view`** — variant-based iterator that lazily concatenates multiple ranges of a common reference type; supports input/forward/bidirectional iteration and `std::ranges::view_interface` helpers
- **`util::concat`** — factory function wrapping each range via `std::views::all` so temporaries are handled safely; works with pipe composition

```cpp
std::vector<int> a{1, 2, 3};
std::array<int, 2> b{4, 5};

for (int x : util::concat(a, b)) { ... }  // 1 2 3 4 5

auto evens = util::concat(a, b)
           | std::views::filter([](int x){ return x % 2 == 0; });
```

> ## 2026-04-15 `OrthographicAxis2D` Volume/Layer ports and `PlotCamera` fix

Reviewer: ```suggestion
## 2026-04-15 `OrthographicAxis2D` Volume/Layer ports
```

> - **`OrthographicAxis2D`** processor now exposes optional `MeshInport", `VolumeInport`, and `LayerInport` so axis ranges can be driven directly from a Mesh, Volume, or Layer data extents

Reviewer: ```suggestion
- **`OrthographicAxis2D`** processor now exposes optional `MeshInport`, `VolumeInport`, and `LayerInport` so axis ranges can be driven directly from a Mesh, Volume, or Layer data extents
```

> ## 2026-04-14 Scientific color maps added to utilities, and to TF presets menus

Reviewer: ```suggestion
## 2026-04-14 Scientific color maps added to utilities and the TF presets menus
```

> A large collection of perceptually uniform and scientifically accurate color maps has been added to the Base module, expanding the set of built-in transfer functions available in Inviwo.

Reviewer: ```suggestion
A large collection of perceptually uniform scientific color maps created by Fabio Crameri ([fabiocrameri.ch/colourmaps/](https://fabiocrameri.ch/colourmaps/)) has been added to the Base module, expanding the set of built-in transfer functions available in Inviwo.
```
